CHANGELOG.rst

```diff
@@ -2,11 +2,34 @@
 Changelog
 =========
 
-Upgrade notes
--------------
+2.7.10 (2024-11-22)
+===================
+
+Upgrade procedure
+-----------------
+
+* ⚠️ Ensure that the form variables initial values for selectboxes components match with
+  their respective component default values.
+  
+  After upgrading to 2.7.10, run the ``app/bin/fix_selectboxes_component_default_values.py``
```

Comment:
```suggestion
  After upgrading to 2.7.10, run the ``/app/bin/fix_selectboxes_component_default_values.py``
```
I think the working directory by default is `/app`, so using absolute paths is most robust.